src/qt/receivecoinsdialog.h

```diff
@@ -54,6 +54,9 @@ public Q_SLOTS:
     GUIUtil::TableViewLastColumnResizingFixer *columnResizingFixer;
     WalletModel *model;
     QMenu *contextMenu;
+    QAction *copyLabelAction;
+    QAction *copyMessageAction;
+    QAction *copyAmountAction;
```

Comment:
style nit (see [clang-format-diff.py](https://github.com/bitcoin/bitcoin/tree/master/contrib/devtools#clang-format-diffpy)):
```suggestion
    QAction* copyLabelAction;
    QAction* copyMessageAction;
    QAction* copyAmountAction;
```

Reply:
Addressed in 3f83f05f52be8f3bac2267d447b5037409cde17d